conjur/logout/logout_logic.py

```diff
@@ -17,8 +17,8 @@ class LogoutLogic:
     def __init__(self, credentials):
         self.credentials = credentials
 
-    def remove_credentials(self, conjurrc_appliance_url):
+    def remove(self, conjurrc_appliance_url):
```

Comment:
i think privious name is better. Remove without anything dont give me much information